lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -17,6 +17,14 @@ def fulfillments(options = {})
       fulfillment_hashes.map { |fulfillment_hash| Fulfillment.new(fulfillment_hash) }
     end
 
+    def locations_for_move(options = {})
```

Comment:
What could be passed in as `options`? Could add a test for it.

Reply:
Actually, there is no `options`. I will remove it.